server/app/services/logging.rb

```diff
@@ -3,7 +3,7 @@
 module Logging
 
   def self.initialize_logger(log_target = STDOUT, log_level = Logger::INFO)
-    @logger = Logger.new(log_target)
+    @logger = Logger.new(ENV['RACK_ENV'] == 'test' ? StringIO.new : log_target)
```

Comment:
I suppose they're coming from websocketbackend or something is reinitializing the Logging thing during tests without the LOG_TARGET.